lib/discordrb/data/user.rb

```diff
@@ -172,6 +159,24 @@ def webhook?
       end
     end
 
+    # @return [String, nil] the game the user is currently playing, or `nil` if none is being played.
+    # @deprecated Please use {ActivitySet#game} for information about a users game activity
+    def game
+      @activities.game
+    end
+
+    # @return [Integer] returns 1 for twitch streams, 0 for no stream.
+    # @deprecated Please use {ActivitySet#streaming} to check for the existance of a stream
```

Comment:
Can you make the wording consistent between all three of these methods:
```suggestion
    # @deprecated Please use {ActivitySet#streaming} for information about the users stream activity
```
Also note the language of "the user" (instead of "a user"), and please update `stream_url` and `game` accordingly